openapi.yaml

```diff
@@ -736,18 +736,28 @@ components:
       in: query
       description: |
         Le processus utilisé pour tricoter le tissu, parmi la liste de choix suivants :
-          - mix : Tricotage moyen (mix de métiers circulaire & rectiligne)
-          - fully-fashioned : Tricotage fully-fashioned
-          - seamless : Tricotage seamless
-          - circular : Tricotage circulaire, inventaire désagrégé
-          - straight : Tricotage rectiligne, inventaire désagrégé
+          - `mix` : Tricotage moyen (mix de métiers circulaire & rectiligne)
+          - `fully-fashioned` : Tricotage fully-fashioned / seamless
+          - `integral` : Tricotage intégral / whole garment
+          - `circular` : Tricotage circulaire, inventaire désagrégé
+          - `straight` : Tricotage rectiligne, inventaire désagrégé
 
           **⚠️ Sur les produits tissés, ce paramètre n'a aucun impact.**
       required: false
       style: form
       schema:
         type: string
-        enum: ["mix", "fully-fashioned", "seamless", "circular", "straight"]
+      examples:
+        mix:
+          value: "mix"
+        fully-fashioned:
+          value: "fully-fashioned"
+        integral:
+          value: "integral"
+        circular:
+          value: "circular"
+        straight:
+          value: "straight"
```

Comment:
Ah sorry if it wasn't clear, I was actually suggesting to link to a definition, eg

```yaml
        enum:
            "$ref": "#/components/schemas/KnittingProcessEnum"
```

so the same definition can be used for both the GET param and thr JSON one for POST.

Reply:
I removed the `enum` there, and simply kept it in `knittingProcess`, which does exactly that, right?

Reply:
I'll have to checkout the branch and check, give me a minute

Reply:
So, nope, examples are not acceptable values 
![image](https://github.com/MTES-MCT/ecobalyse/assets/41547/a4361220-002f-4ce4-9366-db8eb22ba97f)